packages/financial-templates-lib/test/price-feed/DefiPulseTotalPriceFeed.js

```diff
@@ -0,0 +1,87 @@
+require("dotenv");
+const { DefiPulseTotalPriceFeed } = require("../../src/price-feed/DefiPulseTotalPriceFeed");
+const { NetworkerMock } = require("../../src/price-feed/NetworkerMock");
+const winston = require("winston");
+var Web3 = require("web3");
+var web3 = new Web3("http://localhost:8545");
+
+contract("DefiPulseTotalPriceFeed.js", function() {
+  let defiPulseTotalPriceFeed;
+  let mockTime = 1611583300;
+  let networker;
+
+  const apiKey = process.env.DEFIPULSE_API_KEY;
```

Comment:
unit tests should not rely on having this env set.

Reply:
Changed to a test string